packages/core/core/src/factories/useUserBillingFactory.ts

```diff
@@ -0,0 +1,96 @@
+import { Ref, computed } from '@vue/composition-api';
+import { UseUserBilling } from '../types';
+import { sharedRef } from '../utils';
+
+export interface UseUserBillingFactoryParams<ADDRESS> {
+  addAddress: (address: ADDRESS) => Promise<ADDRESS[]>;
+  deleteAddress: (address: ADDRESS) => Promise<ADDRESS[]>;
+  updateAddress: (address: ADDRESS) => Promise<ADDRESS[]>;
+  load: () => Promise<ADDRESS[]>;
+  setDefault: (address: ADDRESS) => Promise<void>;
```

Comment:
you should return the new default address so in the factory its picked up and assigned to a variable